Frontend/ui-driver/src/Screens/DriverProfileScreen/ScreenData.purs

```diff
@@ -66,19 +89,47 @@ type Listtype =
 optionList :: String -> Array Listtype
 optionList dummy =
     [
-      {menuOptions: DRIVER_PRESONAL_DETAILS , icon:"ny_ic_profile,https://assets.juspay.in/nammayatri/images/driver/ny_ic_profile.png"},
-      {menuOptions: DRIVER_VEHICLE_DETAILS , icon:"ny_ic_car_profile,https://assets.juspay.in/nammayatri/images/driver/ny_ic_car_profile.png"}
+      -- {menuOptions: DRIVER_PRESONAL_DETAILS , icon:"ny_ic_profile,https://assets.juspay.in/nammayatri/images/driver/ny_ic_profile.png"},
```

Comment:
If this isn't required, can you remove it? If req in future, then add proper message